### ISSUE TITLE
New helm window popup conflicts

### ISSUE THREAD

**Issue by mkcode** [syl20bnr/spacemacs#1396]
The code introduced in https://github.com/syl20bnr/spacemacs/commit/bfe99322f7082466e24fed99091367374f9bf8b9 - #1360 breaks with my current window config. 

I have this in my config:
`(setq display-buffer-base-action '((display-buffer-use-some-window)))`

What that does for me is reuses my last 'other-window' for commands like `dired-other-window` (`o` in dired-mode by default) and `magit-status`. If I don't have this config set, these 2 commands split my current window, which I do not like. And if I do, helm shows up in my 'other-window' as well as on the bottom.

Any suggestions for making these compatible, and/or alternatively adopting this style as the default behavior?

edit: updated config.

**Comment by mkcode** [comment]
@tudho - Hoping you can help with this.

**Comment by tuhdo** [comment]
How do I reproduce this? I do it like this and it does not occur to me:

- Set the above config.
- Open a Dired buffer.
- `C-x 4 d` open another Dired buffer to have other window.
- `C-x 4 d` again expecting it to reuse the previous other window.

So far `C-x 4 d` and it shows a Helm window at the bottom, nothing is broken.

**Comment by mkcode** [comment]
Apologies, I changed the config when I made created this issue:

The config I have is:
`(setq display-buffer-base-action '((display-buffer-use-some-window))`

To reproduce:

 * Open a dired buffer
 * `SPC w /` - create a vertical window split
 * `SPC w w` - back to the dired buffer
 * `o` (on a file in dired) should open a the file in the new window
 * `SPC p h` or any other helm command will show it in both the other window and below now.

@tuhdo - Thanks. very much appreciate the help with this.
